cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/CgmesNamingStrategyTest.java

```diff
@@ -41,23 +37,6 @@
  */
 class CgmesNamingStrategyTest extends AbstractSerDeTest {
 
-    @Test
```

Comment:
Shouldn't we keep an equivalent of the deleted tests, but in another CIM version?

Reply:
A test with `NamingStrategyFactory.CGMES` has been kept (`testExportUsingCgmesNamingStrategyMicroGrid()`). It was deemed complete enough to test the naming strategy.